2023-ECP/JupyterNotebook/README.md

```diff
@@ -0,0 +1,19 @@
+To run the AWS tutorial, visit https://tutorial.flux-framework.org 
+You can use any login you want, but choose something relatvely uncommon 
+(like your email address) or you may end up sharing a JupyterLab 
+instance with another user. The tutorial password will be provided to you. 
+
+If you have Docker available, you can build and run the tutorial with:
+
+```
+docker build --build-arg GLOBAL_PASSWORD=<password of your choice> -t <desired image name> -f Dockerfile.hub .
+docker build -t <desired image name> -f Dockerfile.spawn .
```

Comment:
It should clear that <desired image name> here is different from the one above. They are for two different images and thus two different names. We can change it to for example, `<desired_image_name_for_spawner>`